### ISSUE TITLE
Change Outlets to Switches

### ISSUE THREAD

**Issue by SmolSoftBoi** [Droccal/homebridge-lay-z-spa#7]
It would appear better in HomeKit if they were Switches instead of Outlets. I'm happy to submit a pull request if you think this would be considered.

**Comment by Mischa1610** [comment]
Hi @SmolSoftBoi, if I understood it correctly @Droccal is not planning to continue to work on this plugin, see his comment on this issue: https://github.com/Droccal/homebridge-lay-z-spa/issues/10#issuecomment-1875836776

@SmolSoftBoi maybe you want to pick this plugin up? Or help me to keep it "alive" (small improvements), but I am not really experienced in Homebridge plugin development (but node & JS/TS development in general), see here my comment: https://github.com/Droccal/homebridge-lay-z-spa/issues/10#issuecomment-2214694058

**Comment by Droccal** [comment]
Hey guys,

I will try to come up with some solutions to all reported problems, just give me some time as I am on vacation right now. I would assume at the end of the month I can hopefully fix all reported findings!

**Comment by Mischa1610** [comment]
> Hey guys,
> 
> I will try to come up with some solutions to all reported problems, just give me some time as I am on vacation right now. I would assume at the end of the month I can hopefully fix all reported findings!

Yes for sure @Droccal, take your time, I just wanted to make sure that this plugin stays active because I also just set it up and plan to use it. So if you need support in the form of PR or something let me know.

Thanks for your commitment and work and happy to see the fixes soon ;-)
